test/hotspot/jtreg/vmTestbase/nsk/jvmti/scenarios/hotswap/HS201/hs201t002/hs201t002.cpp

```diff
@@ -328,11 +320,16 @@ callbackException(jvmtiEnv *jvmti_env, JNIEnv* jni_env, jthread thread,
 
     className = getClassName(jvmti_env, jni_env, exception);
 
-    if (strcmp(EXPECTED_CLASS_SIGN, className) == 0) {
+    if (className != NULL && strcmp(EXPECTED_CLASS_SIGN, className) == 0) {
         jclass klass;
 
-        NSK_DISPLAY2("\n\n>>>> Exception %s in thread - %s\n",
```

Comment:
getClassName() and getThreadName() used the same global buffer to return results. So thread name overrides class name and in log we see:
>>>> Exception hs201t002Thread in thread - hs201t002Thread
instead of 
>>>> Exception Lnsk/jvmti/scenarios/hotswap/HS201/hs201t002a; in thread - hs201t002Thread